app/domain/repositories/ExpensesRepository.scala

```diff
@@ -0,0 +1,13 @@
+package domain.repositories
+
+import domain.entity.Expenses
+import domain.repositories.base._
+
+trait ExpensesRepository
+  extends BaseGetAllRepository[Expenses]
+    with BaseGetOneRepository[Expenses]
+    with BaseUpdateRepository[Expenses]
+    with BaseSaveRepository[Expenses]
+    with BaseDeleteRepository[Expenses]{
```

Comment:
@Bukarachka it's better to remove Boolean in case of delete, as all object is not valid anymore